business_rules/operators.py

```diff
@@ -576,13 +579,33 @@ def empty(self, other_value: dict):
         self.value[f"result_{uuid4()}"] = results
         return True in results
 
```

Comment:
I don't like the loop here. My idea was to group the dataframe by comparator and perform the validation for each grouped part.
What do you guys thin, how can we avoid the loop?